assets/src/utils/charts.ts

```diff
@@ -3,19 +3,26 @@
 import uPlot, { AlignedData } from "uplot";
 import { getXTicksConfig } from "../utils/chartsXTicksConfig";
 
+const metricToTitle: { [metric: string]: string } = {
```

Comment:
```suggestion
const metricToTitle: Record<string, string> = {
```

type metric = "caps" | "event" | "store" | "take_and_demand";
const metricToTitle: Record<metric, string> = {
```